sample.env

```diff
@@ -0,0 +1,3 @@
+#REACT_APP_BASE_URI=http://164.90.247.198
+REACT_APP_BASE_URI=http://localhost:3000
+REACT_APP_CHAIN_ID=1
```

Comment:
Note that moving forward, we'll want to make sure that this can support multiple networks - if the user switches from Mainnet to Rinkeby, e.g., this feature should still work with Rinkeby sources and data.

Not sure if you need to change this here, but just something to be aware of.